src/others/snapshot/lv_snapshot.c

```diff
@@ -86,6 +86,9 @@ lv_result_t lv_snapshot_take_to_draw_buf(lv_obj_t * obj, lv_color_format_t cf, l
         case LV_COLOR_FORMAT_A8:
         case LV_COLOR_FORMAT_L8:
         case LV_COLOR_FORMAT_I1:
+        case LV_COLOR_FORMAT_ARGB2222:
+        case LV_COLOR_FORMAT_ARGB4444:
+        case LV_COLOR_FORMAT_ARGB1555:
```

Comment:
Software render and GPU renders are diverging quite a bit now. Probably we should add functions like `lv_draw_unit_is_dest_color_format_supported` or so.

Leave it as it is for now, just keep it in mind.